src/lib/api/http.ts

```diff
@@ -14,6 +14,10 @@ import { API_HOST } from '../../lib/api';
 
 const debug = debugLib( '@automattic/vip:http' );
 
+type FetchOptions = Omit<RequestInit, 'body'> & {
+	body?: BodyInit | Record<string, unknown>;
+}
```

Comment:
Sometimes, we pass objects here. They will be encoded into strings, but we need a proper type definition because `RequestInit` does not like `body` as an `object`.